playground/frontend/lib/services/analytics/events/new_example.dart

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:playground_components/playground_components.dart';
+
+import 'constants.dart';
+
+/// 'New Example' button is clicked in the app header.
+class NewExampleAnalyticsEvent extends AnalyticsEvent {
```

Comment:
It would be great to see more test coverage for this.  Like other extensions of AnalyticsEvent seemingly just data classes, it would be great to assert that the values are what is expected (I see the name property for example). In this context I would like to see a test mocking New Example button clicked and that the assertion passes that the name is what it should be.